chain/walker.go

```diff
@@ -78,7 +80,7 @@ func (c *Walker) WalkChain(ctx context.Context, node lens.API, ts *types.TipSet)
 	}
```

Comment:
The call above on line 76/78 is basically a noop. All it does now is set the indexer's last tipset to the provided tipset, right?